### PR TITLE
Feature/css

### PR DESCRIPTION
**Pull request by sydnerd**
### Acceptance Criteria
As a user, you can see a pretty applicaiton

### Issues Resolved
Added CSS
### Code Reviewers
@alexmkio

### CODE REVIEW

File: src/components/eventCard/EventCard.js [hunk 1]
```diff
@@ -20,19 +20,19 @@ export const EventCard = ({ event }) => {
...
-        <dd>{event.dog.name}</dd>
-
-        <dt>Event Name</dt>
-        <dd>
-          <Link to={`/event/${event.id}`} key={event.id} >{event.name}</Link>
-        </dd>
-
-        <dt>Event Date</dt>
-        <dd>{(new Date(event.eventDatetime)).toLocaleString()}</dd>
+        <div className='description-container'>
```
Comment (alexmkio) [inline src/components/eventCard/EventCard.js:23]: what are these divs accomplishing? Could the styling be applied to a different semantic element; i.e. the dl, dt, or dd?

File: src/components/header/Header.js [hunk 2]
```diff
@@ -4,12 +4,31 @@ import { Link } from 'react-router-dom'
...
+      <section className='header-container'>
+        <h1 className='title'>Pawty Trainer</h1>
+        {/* <p className='material-icons'>pets</p> */}
+      </section>
       <nav>
-        <Link to ='/'>Dashboard</Link>
-        <Link to ='/create'>Add Dog</Link>
-        <Link to ='/add_event'>Add Event</Link>
-        <Link to ='/calendar'>Calendar</Link>
+        <Link
```
Comment (alexmkio) [inline src/components/header/Header.js:12]: This looks wayyy less readable to me. Not necessary because those lines aren't exceeding 80 characters.

**Review by alexmkio — CHANGES_REQUESTED** [review]

File: src/components/addEvent/AddEvent.js [hunk 3]
```diff
@@ -41,15 +41,15 @@ export const AddEvent = ({ dogs }) => {
   return (
     <>
       <form>
-        <h1>Add New Event</h1>
+        <h1 className='text'>Add New Event</h1>
```
[the remark this answers is not in the document]
Comment (sydnerd) [reply src/components/addEvent/AddEvent.js:44]: changed

File: src/components/header/Header.js (hunk 2 above)
Comment (sydnerd) [reply src/components/header/Header.js:12]: sorry, i was playing around with adding a classname. ill change it back.

File: src/components/eventCard/EventCard.js (hunk 1 above)
Comment (sydnerd) [reply src/components/eventCard/EventCard.js:23]: For styling, when you use the other tags, it messes it up

**Pull request merged**